Software/src/SoundManager.hpp

```diff
@@ -44,19 +43,22 @@ class SoundManager : public QObject
 	Q_OBJECT
 
 public:
-	explicit SoundManager(int hWnd, QObject *parent = 0);
-	~SoundManager();
+	SoundManager(QObject *parent = 0);
+	virtual ~SoundManager();
+	static SoundManager* create(int hWnd = 0, QObject* parent = 0);
 
 signals:
 	void updateLedsColors(const QList<QRgb> & colors);
 	void deviceList(const QList<SoundManagerDeviceInfo> & devices, int recommended);
 
 public:
-	void start(bool isMoodLampEnabled);
+	virtual void start(bool isMoodLampEnabled) { Q_UNUSED(isMoodLampEnabled); Q_ASSERT(("Not implemented", false)); };
```

Comment:
Please call this class SoundManagerBase. As opposed to the GrabManager it's not meant to be instantiated. And rename the bool to something like started.

Reply:
agreed
>And rename the bool to something like started.

it was like this when I got here, I swear!